LibraBFT/Impl/Consensus/RoundManager/Properties.agda

```diff
@@ -309,47 +309,50 @@ module ensureRoundAndSyncUpMSpec
 
   open ensureRoundAndSyncUpM now messageRound syncInfo author helpRemote
 
-  -- TODO-2: Add requirements capturing that any signatures in any QCs in the SyncInfo have been
-  -- sent before.  Probably want to refactor properties in LibraBFT.Impl.Properties.Util.QC
-  postulate
-    Requirements : SentMessages → SyncInfo → Set
-
-  module _ (pool : SentMessages) (pre : RoundManager) (reqs : Requirements pool syncInfo) where
+  module _ (pool : SentMessages) (pre : RoundManager) where
 
     record Contract (r : Either ErrLog Bool) (post : RoundManager) (outs : List Output) : Set where
       constructor mkContract
       field
         -- General invariants / properties
-        rmInv         : Preserves (RoundManagerInv pool) pre post
+        rmInv         : Preserves RoundManagerInv pre post
         noEpochChange : NoEpochChange pre post
         noVoteOuts    : OutputProps.NoVotes outs
         -- Voting
         noVote        : VoteNotGenerated pre post true
+        -- Signatures
+        outQcs∈RM     : QCProps.OutputQc∈RmOrMsg outs post {!!}
```

Comment:
I think this should be a similar property that requires either the `QC` is in the `RoundManager` or  it is in `SyncInfo`.  We know at call sites that the `SyncInfo` has come from a message that has been sent.